lib/flood_risk_engine/engine.rb

```diff
@@ -12,12 +12,35 @@ class Engine < ::Rails::Engine
       app.config.i18n.load_path += Dir[config.root.join("config/locales/**/**/", "*.{rb,yml}").to_s]
     end
 
+    # Automatically add our migrations into the main apps migrations
```

Comment:
Agreed. Done.  No, removed, just states that it exports them automatically to app